assets/stylesheets/_components.scss

```diff
@@ -266,9 +266,7 @@
 @import 'components/version/style';
 @import 'components/vertical-nav/style';
 @import 'components/vertical-nav/item/style';
-@import 'components/vertical-menu/style';
```

Comment:
did you mean to drop this one?

Reply:
@blowery You're right.  Must have been accidentally branched from my the vertical menu styling migration branch.  Amended this.